src/mem3/src/mem3_util.erl

```diff
@@ -509,6 +507,59 @@ sort_ranges_fun({B1, _}, {B2, _}) ->
     B1 =< B2.
 
 
+get_or_create_db(DbName, Options) ->
+    case couch_db:open_int(DbName, Options) of
+        {ok, _} = OkDb ->
+            OkDb;
+        {not_found, no_db_file} ->
+            try
+                DbOpts = case mem3:dbname(DbName) of
+                    DbName  -> [];
+                    MDbName -> mem3_shards:opts_for_db(MDbName)
+                end,
+                %% Options1 = opts_merge(DbOpts, Options)
```

Comment:
Commented out code?

Reply:
Yeah, I mentioned that in the PR notes. I'm not a fan of this function, so I left it commented out and just merged the lists in the hopes of finding a better approach.

Reply:
Ah, didn't realize there was commentary in the comments area. 😃